WordPress/src/main/java/org/wordpress/android/ui/sitecreation/verticals/NewSiteCreationVerticalsViewModel.kt

```diff
@@ -269,14 +280,14 @@ class NewSiteCreationVerticalsViewModel @Inject constructor(
     }
 
     sealed class VerticalsListItemUiState {
+        lateinit var onItemTapped: () -> Unit
```

Comment:
I feel like using `lateinit` for this is kind of error prone. I'd personally prefer either `nullable` or a different approach.

Reply:
Good point! Using a nullable type with a unittest which checks it's not null is safer.